pkg/apis/autoscaling/v2beta2/defaults.go

```diff
@@ -18,11 +18,51 @@ package v2beta2
 
 import (
 	autoscalingv2beta2 "k8s.io/api/autoscaling/v2beta2"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/apis/autoscaling"
 )
 
+var (
+	scaleUpLimitPercent         int32 = 100
+	scaleUpLimitMinimumPods     int32 = 4
+	scaleUpPeriod               int32 = 60
```

Comment:
This should really be 15 to keep parity with existing defaults, which resync HPAs every 15 seconds.